.github/workflows/sync-to-aur.yml

```diff
@@ -0,0 +1,45 @@
+name: Sync package to AUR
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save ssh key
+        run: |
+          mkdir -p ~/.ssh
+          echo ${{ secrets.AUR_SSH_KEY }} > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+
+      - name: Install and configure git
+        run: |
+          sudo apt-get install git
+          git config --global user.email ${{ secrets.GIT_COMMIT_EMAIL }}
+          git config --global user.name ${{ secrets.GIT_COMMIT_NAME }}
+
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Sync packages to AUR
+        run: |
+          set -x
+          for i in $(git show --pretty="" --name-only | cut -d/ -f1 | uniq)
```

Comment:
Is it guaranteed to be sorted? If not, `sort -u` to the rescue.

Reply:
I think it's sorted. But using `sort -u` for sure anyway.